modules/consensus/synchronize.go

```diff
@@ -76,13 +81,24 @@ func (cs *ConsensusSet) threadedReceiveBlocks(conn modules.PeerConn) error {
 		return err
 	}
 
+	// Broadcast the last block accepted. This functionality is in a defer to
+	// ensure that a block is always broadcast if any blocks are accepted. This
+	// is to stop an attacker from preventing block broadcasts.
+	chainExtended := false
+	defer func() {
+		if chainExtended {
+			currentBlock := cs.CurrentBlock()
```

Comment:
Potential bug... If the last block sent by the `SendBlocks` RPC doesn't extend the longest chain, shouldn't we still broadcast the block?